committeeoversightapp/views.py

```diff
@@ -135,46 +171,28 @@ def prepare_results(self, qs):
                             kwargs={'pk':item.pk}
                         ))
                     ),
-                ])
-        else:
-            for item in qs:
-                json_data.append([
-                    item.updated_at.strftime("%Y-%m-%d %I:%M%p %Z"),
-                    detail_string.format(
-                        escape(reverse_lazy(
-                            'detail-event',
-                            kwargs={'pk':item.pk}
-                        )),
-                        escape(item.name.title())
-                    ),
-                    item.start_date,
-                    '',
-                    '',
-                ])
+                ]
+                json_data.append(row_data)
+            # if not authenticated, only include display hearing categories 
+            elif not item.category or item.category.name in settings.DISPLAY_CATEGORIES:
```

Comment:
This is a little confusing to me. If there is not an item category, or if the category is in display categories, add two empty values?

Reply:
`category_string` will be an empty value if there is not a category, and it is added to the row unconditionally in L156. Will this introduce a redundant empty value?

Reply:
I think my refactor addresses this question. I also realized that I'm not actually sure if hearings without a category should be displayed to non-admin users. I opened a question issue for it (#134) and emailed Jamie. For now, hearings with no category will only show up for admin users.